pallets/pallet-did-lookup/src/lib.rs

```diff
@@ -131,6 +131,11 @@ pub mod pallet {
 
 		/// Migration manager to handle new created entries
 		type BalanceMigrationManager: BalanceMigrationManager<AccountIdOf<Self>, BalanceOf<Self, I>>;
+
+		// Flag specifying whether there should only ever be a single account <-> DID
+		// link, or multiple.
+		#[pallet::constant]
+		type UniqueLinkingEnabled: Get<bool>;
```

Comment:
I made this a `Get<bool>` instead of a const to both allow clients to query whether this pallet enforces unique links or not, and to also make testing easier since this value can be dynamically mocked, as opposed to a const.